web/app/models.py

```diff
@@ -199,14 +199,16 @@ class Schedule(db.Model):
     """Database model for schedules."""
     __tablename__ = 'schedules'
     id = db.Column(db.Integer, primary_key=True)
-    approved = db.Column(db.Boolean)
+    approved = db.Column(db.Boolean, default=False)
     user_id = db.Column(db.Integer, db.ForeignKey("users.id"))
     work_period_id = db.Column(db.Integer, db.ForeignKey("work_periods.id"))
     deviations = db.relationship('Deviation', backref='schedule', lazy='dynamic')
     base_schedule = db.relationship('BaseSchedule', backref='schedule', lazy='dynamic')
 
-    def __init__(self):
-        pass
+    def __init__(self, user_id, work_period_id, approved=False):
```

Comment:
A bit unusual to override the __init__ created by SQLAlchemy. Using positional arguments makes the code less clear.